firmware/robot2015/src-ctrl/scripts/linker/sections_FLASH.ld

```diff
@@ -0,0 +1,235 @@
+/* Linker script for mbed LPC1768 */
```

Comment:
What does this linker script do beyond the default one for mbed?  Is this for the parameter feature?